diff --git a/config.def.h b/config.def.h
index 482901e..7fd9216 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,11 +1,7 @@
-/* See LICENSE file for copyright and license details. */
-
-/*
- * appearance
- *
- * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
+/* St: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
+ * 
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Hack:pixelsize=20:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -16,7 +12,7 @@ static int borderpx = 2;
  * 4: value of shell in /etc/passwd
  * 5: value of shell in config.h
  */
-static char *shell = "/bin/sh";
+static char *shell = "/bin/zsh";
 char *utmp = NULL;
 char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 
